src/main/java/seedu/address/model/client/CountryMatchesInputCountryPredicate.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.model.client;
+
+import java.util.function.Predicate;
+
+import seedu.address.model.country.Country;
+
+/**
+ * Tests that a {@code Client}'s {@code Country} matches the given Country.
+ */
+public class CountryMatchesInputCountryPredicate implements Predicate<Client> {
```

Comment:
Might want to change the predicate name to ClientCountryMatchesInputCountryPredicate for clarity?

Reply:
Yep